cdap-common/src/test/java/co/cask/cdap/common/namespace/SimpleNamespaceQueryAdmin.java

```diff
@@ -14,9 +14,8 @@
  * the License.
  */
 
-package co.cask.cdap.data2.util.hbase;
+package co.cask.cdap.common.namespace;
```

Comment:
We moved this class because SimpleNamespaceQueryAdmin is class which can be used in tests across cdap so keeping it in commons makes more sense than data-fabric.